hack/toolchain.sh

```diff
@@ -7,14 +7,17 @@ trap "rm -rf $TEMP_DIR" EXIT
 
 main() {
     tools
-    kubebuilder
+    if ! command -v kubebuilder &>/dev/null; then
```

Comment:
I'd probably leave this out -- just incase we bump the kb version.

Reply:
Should we change this to check if a new enough version of kubebuilder is installed?